org-cyf-piscine/content/piscine/project-guidance/index.md

```diff
@@ -0,0 +1,75 @@
+   We should not give them step-by-step instructions for everything they need to do - they need to work some things out for themselves.
+1. To practice the skills learnt in ITP, gain experience, and see if the trainee is interested in doing more.
+1. To produce examples of well-done work which could be included in a **portfolio**.
+
+Some projects will be done in groups, and others as individuals. If your project is better suited to one or the other, say so.
+
+## When writing up the project
+
+### Structure
+
+Your project should at least list an introduction and a list of requirements. Additional sections are up to you.
+
+Make sure to include a rubric. TODO: Work out how much we want rubrics to be exposed to trainees vs kept for assessors.
+
+### Make clear what the focus areas are and aren't
+
+Is your project about making a clean UI? Is your project actually about making one particular widget and you don't care about the surrounding UI at all? Is your project really about processing data and just outputting an unstyled &lt;ul&gt; is actually fine? Be explicit about what is expected.
+
+Trainees will waste days building a pretty UI if you don't tell them not to. Don't make them waste their time if a UI isn't important.
+
+Also, reflect this in the rubric: If they have completed everything else, it's fine if they spent some extra time making it pretty. But if you told them not to worry about a UI, and they spent time making a pretty UI but _didn't_ complete required features, they have failed.
+
+### Actively consider what you want to _explain_ and what you want trainees to _work out_
+
+It's ok to reference terms without explaining them and leaving the trainees to Google them.
+
+It's also ok to link to a useful explanation that you recommend.
+
+It's also ok to fully explain something in your project.
+
+Consider which is most appropriate for the things you're trying to teach and test.
+
+### Minimise, but don't completely avoid, ambiguity
+
+At this stage of a trainee's education, it's useful to see how trainees respond to ambiguity, but we don't have enough contact time with them to really explore ambiguity and discuss trade-offs.
+
+Accordingly, try to be specific about anything that may be relevant in an assessment rubric. It's ok to leave things unspecified, but if we're going to be assessing on something, we should be explicit in the requirements what we expect.
+
+When trainees fail, we need to tell them _why_ they failed a project. Don't create opportunities for trainees to be surprised or disappointed by those reasons.
+
+### Write a rubric
+
+The outcomes of these projects will determine who we support to continue learning and get jobs. This affects people's lives. We need to be fair in our decisions.
+
+Rubrics should be as objective as we can make them. Ideally they contain a series of boolean statements that can be easily tested.
```

Comment:
```suggestion
Rubrics should be as objective as we can make them. Ideally they contain a series of boolean statements that can be easily tested without significant time with individual trainees.
```

I'm trying to push back against a criteria like "Understands how X aspect of the code works" - in a code walkthrough this is (relatively) easily testable, but I think this violates our constraint about not investing a lot of volunteer time.